PrimeRust/solution_1/src/main.rs

```diff
@@ -1,18 +1,28 @@
 use primes::{
     print_results_stderr, report_results_stdout, FlagStorage, FlagStorageBitVector,
-    FlagStorageByteVector, FlagStorageBitVectorRotate, FlagStorageBitVectorStriped, PrimeSieve,
+    FlagStorageBitVectorRotate, FlagStorageBitVectorStriped, FlagStorageByteVector, PrimeSieve,
+};
+
+use std::{
+    thread,
+    time::{Duration, Instant},
 };
-use std::{thread, time::{Duration, Instant}};
 use structopt::StructOpt;
 
 pub mod primes {
     use std::{collections::HashMap, time::Duration, usize};
 
+    /// Shorthand for the `u8` bit count to avoid additional conversions.
+    const U8_BITS: usize = u8::BITS as usize;
```

Comment:
It looks like this was just added to stable in 1.53 -- and that reminds me of why I had it hard coded before :) 

We'll need to bump the version of Rust in the Docker file to 1.53 so it builds successfully.

Reply:
You are right - I forgot all about that, good catch.

Well, I don't really know how to attribute commits to different PRs after the fact - should I refork my branch from the main repo and reapply the commits from there?

Reply:
There's a way to cherry pick commits, but I don't know how to do it either :)

I think you should have permissions to push to a branch in this repo as you're a contributor. The easiest might just be to clone this repo, branch from `rust-idiomatic-improvements`, copy in the main.rs from here, etc. 

Going from the main repo should be fine too if that's easier :)